scripts/gen_dlaf_strong-mc.py

```diff
@@ -0,0 +1,143 @@
+#!/usr/bin/env python3
+
+#
+# Distributed Linear Algebra with Future (DLAF)
+#
+# Copyright (c) 2018-2023, ETH Zurich
+# All rights reserved.
+#
+# Please, refer to the LICENSE file in the root directory.
+# SPDX-License-Identifier: BSD-3-Clause
+#
+
+# This file is an example on how to use the miniapp module.
+# Please do not add gen scripts used for benchmarks into the source repository,
+# they should be kept with the result produced.
+
+import argparse
+import miniapps as mp
+import systems
+
+system = systems.cscs["daint-mc"]
+
+dlafpath = "<path_to_dlaf_build_dir>"
+
+run_dir = f"~/ws/runs/strong"
```

Comment:
I guess we can leave out the `f`:
```suggestion
run_dir = "~/ws/runs/strong"
```
?